tests/unit/common/config/test_user_config_cache_bust_lockdown_adversarial.py

```
@@ -93,7 +93,7 @@ def test_cache_bust_rejected_with_every_non_chat_endpoint_type(
     endpoint_type: EndpointType, target: CacheBustTarget
 ) -> None:
     """Every non-chat/responses endpoint paired with a non-NONE cache_bust target raises."""
-    with pytest.raises(ValueError, match="chat or responses"):
+    with pytest.raises(ValueError, match="not supported|chat or responses"):
```

Ruff (0.16.0)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)